movies/src/components/movieCard/MovieCard.js

```diff
@@ -4,9 +4,10 @@ import MovieMenu from '../movieMenu/MovieMenu';
 import MovieDeletePopup from '../movieDeletePopup/movieDeletePopup';
 import MovieEditPopup from '../movieEditPopup/movieEditPopup';
 
+
 export default function MovieCard(props) {
```

Comment:
`deleteMoviePopup` and `editMoviePopup` can be wrapped with `useCallback`

Reply:
fixed